src/retry/BackoffFunction.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This is muddying the PR. Can we keep this cosmetic stuff separate please?

Reply:
It was an intentional decision since almost all of the files were being touched anyways.

In the future will keep in a separate PR!